config.rb

```diff
@@ -5,11 +5,7 @@
 ###
 
 set :markdown_engine, :redcarpet
-set :markdown,
-  fenced_code_blocks: true,
-  smartypants: true,
-  with_toc_data: true,
-  renderer: TechDocsHTMLRenderer
+set :markdown, renderer: TechDocsHTMLRenderer
```

Comment:
I _think_ you can pass it an instantiated one.

```
set :markdown, renderer: TechDocsHTMLRenderer.new(fenced_code_blocks: true)
```

Not sure if that tickles your fancy.